### PR TITLE
[Node] - pnpmVersion as option added

### PR DESCRIPTION
**Pull request by gauravsaini04**
**Feature name**:
 
 * Node
 
 **Description**:
 
 This PR introduces the following functionality :
 
 * `pnpnVersion` is added as an option as it has the compatibility requirements with specific node versions
 
 _Changelog_:
 
 * Updated `install.sh`
 * Updated tests to validate the above
 
 **Checklist**:
 
 * [x]   Checked that applied changes work as expected

### CODE REVIEW

File: src/node/devcontainer-feature.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "id": "node",
-    "version": "1.4.1",
+    "version": "1.4.2",
```
Comment (samruddhikhandale) [inline src/node/devcontainer-feature.json:3]:
```suggestion
    "version": "1.5.0",
```

File: src/node/install.sh [hunk 2]
```diff
@@ -16,6 +16,9 @@ INSTALL_TOOLS_FOR_NODE_GYP="${NODEGYPDEPENDENCIES:-true}"
 # alongside NODE_VERSION, but not set as default.
 ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS:-""}"
 
+#pnpm Version
```
Comment (samruddhikhandale) [inline src/node/install.sh:19]: remove?

File: src/node/install.sh [hunk 3]
```diff
@@ -16,6 +16,9 @@ INSTALL_TOOLS_FOR_NODE_GYP="${NODEGYPDEPENDENCIES:-true}"
...
+PNPM_VERSION="${PNPMVERSION:-"7"}"
```
Comment (samruddhikhandale) [inline src/node/install.sh:20]:
```suggestion
PNPM_VERSION="${PNPMVERSION:-"latest"}"
```

Let's install latest version when use does not pass a version

File: src/node/install.sh [hunk 4]
```diff
@@ -377,7 +380,7 @@ else
             [ ! -z "$http_proxy" ] && npm set proxy="$http_proxy"
             [ ! -z "$https_proxy" ] && npm set https-proxy="$https_proxy"
             [ ! -z "$no_proxy" ] && npm set noproxy="$no_proxy"
-            npm install -g pnpm
+            npm install -g pnpm@"${PNPM_VERSION}"
```
Comment (samruddhikhandale) [inline src/node/install.sh:383]: Need updates when once you resolve my previous comment

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: src/node/devcontainer-feature.json (hunk 1 above)
Comment (gauravsaini04) [reply src/node/devcontainer-feature.json]: done

File: src/node/install.sh (hunk 3 above)
Comment (gauravsaini04) [reply src/node/install.sh]: done

**Pull request closed without merging**